megamek/src/megamek/common/LandAirMech.java

```diff
@@ -1130,7 +1130,7 @@ public void clearBombChoices() {
     @Override
     public int reduceMPByBombLoad(int t) {
         // bombs don't impact movement
-        return 0;
```

Comment:
That goes against the comment, and doesn't appear related?

Reply:
the movement it not impacted so, so it should return the same value.